Assets/Plugins/Source/Core/EOSConfig.cs

```diff
@@ -120,6 +123,11 @@ static EOSConfig()
         /// </summary>
         public List<string> authScopeOptionsFlags;
 
+        /// <summary>
+        /// Flags used to configure platform integration management flags.
+        /// </summary>
+        public List<string> integratedPlatformManagementFlags;
```

Comment:
I believe this is better separated from EOSConfig because I don't believe there is a meaningful default besides 'disabled'. Keeping it separate also better mirrors how the EOS SDK itself keeps the flags separate.

Reply:
The problem with separating the value from `EOSConfig` proper is that it makes it impossible to make use of the flags on Windows, since the WindowsConfig was not designed to be separate from `EOSConfig`.

In _future_ implementations that will be upcoming, this value and field member will be more appropriately placed. The reason I chose to include this addition to the existing config implementation when the brunt of this PR is to define `RuntimeConfig` is because it made the implicit conversion operator that I defined for that struct complete in that the full set of runtime configuration values are represented within, and can EOSConfig can be converted to and from it effectively.

Reply:
Adding the IntegratedPlatformManagmentFlags to `EOSConfig` now also means that users will have config files with the flags both in their base EOSConfig and in their platform specific flags in the future, if we do a release between now and when that future implementation is done. In other words, we would be doing a field add, then needing to deprecate and migrate the information to the appropriate place in future releases, as apposed to just creating the field in a new place.

Reply:
If any user has the field defined in the other platform file, it isn't being collected and applied by anything anywhere, so it's absence cannot possibly be missed.

Similarly, an extra field in the JSON does not cause an error to be thrown if the corresponding field member does not exist in the object that is being deserialized.

Reply:
I have updated the PR to remove this, putting it back within `PlatformConfig`.